server/ServerFSM.cpp

```diff
@@ -622,6 +622,59 @@ sc::result Idle::react(const Error& msg) {
 }
 
 sc::result Idle::react(const Hostless&) {
+    std::string autostart_load_filename = GetOptionsDB().Get<std::string>("load");
+    if (!autostart_load_filename.empty()) {
```

Comment:
prefer to check that the filename is empty, and if so early exit, rather than having all the following code indented a level with nothing substantial after the indented block

Reply:
Fixed